tyk/README.md

```diff
@@ -0,0 +1,172 @@
+DogStatsD implements the `StatsD` protocol and adds a few Datadog-specific extensions.
+Tyk is using `Histogram metric type`.
+
+Please set up the following Datadog and DogStatsD environment variables in your environment:
+
+| DD Environment variable | Value | Description |
+|---------------------------|-------------|------|
+| DD_API_KEY | {your-datadog-api-key} | For the Datadog Agent to connect the DD portal. Your API key can be found in [Account Settings][22]. |
+| DD_ENV |    tyk-demo-env   |   Sets the environment name. |
+| DD_DOGSTATSD_TAGS | "env:tyk-demo" |  Additional tags to append to all metrics, events, and service checks received by this DogStatsD server. |
+| DD_LOGS_ENABLED | true | Enables log collection for the Datadog Agent. |
+| DD_LOGS_CONFIG_CONTAINER_COLLECT_ALL | true | Collects logs from containers.. |
+| DD_DOGSTATSD_SOCKET | /var/run/docker.sock | Path to the Unix socket to listen to.  Docker compose mounts this path. |
+| DD_DOGSTATSD_ORIGIN_DETECTION | true | Enables container detection and tagging for Unix socket metrics. |
+| DD_DOGSTATSD_NON_LOCAL_TRAFFIC | true | Listens for DogStatsD packets from other containers. (Required to send custom metrics). |
+| DD_AGENT_HOST | dd-agent | Name of the agent host in docker. |
+| DD_AC_EXCLUDE | redis | Excludes Datadog redis checks. (Optional) |
+| DD_CONTAINER_EXCLUDE | true | Excludes docker checks for the Datadog Agent. |
+
+After setting environment variables listed above, set up the agent [with DogstatsD][24]
+
+[Restart the Agent][4] after setup.
+
+### Validation
+
+Create a dashboard (or import [the sample][11]) and add a widget. In the section **Graph your data** under the
+**metric** option, start typing the namespace you chose for the pump in the config `pump.conf` under `dogstatsd.namespace`.
+In the example above it was `tyk`. Once you start typing you will see all the available metrics - `tyk.request_time.avg` etc.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this check.
+
+### Dashboards
+
+With Datadog, you can create dashboards that display statistics about your API services and their consumption.
+
+Here's an example for such a dashboard:
+![Tyk Analytics dashboard example][10]
+
+**Note: You can [import][11] the above dashboard and use it as an example or baseline for your own dashboard.**
+
+
+### Events
+
+The tyk integration does not include any events.
+
+### Service Checks
+
+The tyk integration does not include any service checks.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][8].
+
+
+[1]: https://tyk.io/
+[2]: https://docs.datadoghq.com/agent/kubernetes/integrations/
+[3]: https://tyk.io/wp-content/uploads/2021/07/tyk_logo_no_bg.png
+[4]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
+[5]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/tyk/images/tyk-in-lab.png
```

Comment:
This link is broken/not in the `images` directory.

Reply:
@fanny-jiang I'm not sure how to fix this. If I try my fork it works `https://raw.githubusercontent.com/letzya/integrations-extras/master/tyk/images/tyk-in-lab.png`

Reply:
Can you please help me understand which one isn't in the images directory?